R/makeNhoods.R

```diff
@@ -48,46 +49,57 @@
 #' @importFrom BiocNeighbors findKNN
 #' @importFrom igraph neighbors as_ids
 #' @importFrom stats setNames
-makeNhoods <- function(x, prop=0.1, k=21, d=30, refined=TRUE, reduced_dims="PCA") {
+makeNhoods <- function(x, prop=0.1, k=21, d=30, refined=TRUE, reduced_dims="PCA", seed = NULL) {
     if(is(x, "Milo")){
         message("Checking valid object")
         # check that a graph has been built
-        if(!.valid_graph(graph(x))){
+        if(!.valid_graph(miloR::graph(x))){
             stop("Not a valid Milo object - graph is missing. Please run buildGraph() first.")
         }
-        graph <- graph(x)
-        X_reduced_dims  <- reducedDim(x, reduced_dims)
+        X_graph <- miloR::graph(x)
+        X_reduced_dims  <- SingleCellExperiment::reducedDim(x, reduced_dims)
```

Comment:
for the `graph` function in particular, I've used the `package::function()` style call because the `igraph` and `miloR` versions of `graph` conflict depending on which one is loaded first on the user side.

Reply:
This is only relevant for miloR::graph and igraph::graph specifically, and does not apply to other function calls, i.e. the `SingleCellExperiment::reducedDim()`. We will change the `miloR::graph` slot name in the next major release.

Reply:
yup, removing other instances now!

Reply:
sorry!